config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-3.yaml

```diff
@@ -182,10 +182,10 @@ presubmits:
         resources:
           requests:
             cpu: "1"
-            memory: "800Mi"
+            memory: "1.5Gi"
```

Comment:
the image used is also a problem , it uses go 1.20 which is not compatible with the golangci-lint version we use in v0.3

Reply:
IIUC, we use go 1.19 in release-0.3.

https://github.com/kubernetes-sigs/kueue/blob/5ed20eb8e40b9ca7c3f2f861dd368950ffe33721/go.mod#L3

Reply:
yes sorry